source/declarations.tex

```diff
@@ -4402,6 +4417,48 @@
 The call is used
 to direct-initialize, according to the rules above, the object
 that is the destination of the copy-initialization.
+\item
```

Comment:
I think this should precede bullet 17.6.3, because the preceding bullet has an unqualified "otherwise" that already eats the remainder of the cases. Also, the preceding bullet claims "copy initialization cases", but what we have here is direct-initialization.

Reply:
Done. @jensmaurer, please have a look now to make sure it's what you intended.

Reply:
The new bullet was still unreachable. Massaged more to make it reachable.